code/game/objects/items.dm

```diff
@@ -781,6 +783,10 @@
 /obj/item/proc/item_action_slot_check(slot, mob/user, datum/action/action)
 	if(slot & (ITEM_SLOT_BACKPACK|ITEM_SLOT_LEGCUFFED)) //these aren't true slots, so avoid granting actions there
 		return FALSE
+	if(action_slots)
```

Comment:
You probably want `isnull` unless you are intentionally want `NONE` to also fail

Reply:
Though I guess it'd make sense to do `if(slots == NONE) return FALSE`?

Reply:
> Though I guess it'd make sense to do `if(slots == NONE) return FALSE`?

yeah, NONE should probably always return false